tests/testthat/comparison_data/energy/L117.RsrcCurves_EJ_R_tradbio.csv

```diff
@@ -0,0 +1,60 @@
+# ../energy-data/level1//L117.RsrcCurves_EJ_R_tradbio.csv
+# Written by LA117.tradbio.R
+# Fri Mar 31 15:50:41 2017
+# " Traditional biomass resources by GCAM region "
+# " Unit = EJ "
```

Comment:
Yes, we figured that out and I adjusted some input and output file units, so should be all good.